preinstall.sh

```diff
@@ -22,3 +22,4 @@ fi
 
 git submodule update --init --recursive
 
+cp -r native-core gradleops/AndroidSeed/tealeaf/src/main/jni/core
```

Comment:
syslink wont work ?

Reply:
Something did not work but I think this shell be closed after this is closed.
https://github.com/CodeCombiner/native-android/issues/51
 So I will open new issue.